modules/webxr/webxr_interface_js.cpp

```diff
@@ -415,10 +415,8 @@ void WebXRInterfaceJS::_update_tracker(int p_controller_id) {
 		int *axes = godot_webxr_get_controller_axes(p_controller_id);
 		if (axes) {
 			for (int i = 0; i < axes[0]; i++) {
-				InputDefault::JoyAxis joy_axis;
-				joy_axis.min = -1;
-				joy_axis.value = *((float *)axes + (i + 1));
-				input->joy_axis(p_controller_id + 100, i, joy_axis);
+				float value = *((float *)axes + (i + 1));
+				input->joy_axis(p_controller_id + 100, i, value);
```

Comment:
Worth noting that the code is different in the `master` branch (and thus not changed in #56238), I'm not sure if it means that either the `3.x` or `master` version might be missing a follow-up fix? @dsnopek @Faless 

In `master` it's:
```cpp
                int *axes = godot_webxr_get_controller_axes(p_controller_id);
                if (axes) {
                        // TODO again just a temporary fix, split these between proper float and vector2 inputs
                        for (int i = 0; i < axes[0]; i++) {
                                char name[1024];
                                sprintf(name, "axis_%i", i);

                                float value = *((float *)axes + (i + 1));
                                ;
                                tracker->set_input(name, value);
                        }
                        free(axes);
                }
```